.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+<!-- Try to link to an open issue for more information. -->
+
+<!-- In addition to that please answer these questions: -->
+
+**What kind of change does this PR introduce?**
+
+<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->
+
+**Does this PR introduce a breaking change?**
+
+<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
```

Comment:
What is a 'breaking change'? I think if we add this section, we need to make it clear what we mean by a 'breaking change'. And also what is 'migration path for existing application'?

Reply:
A change in one part of a software system that potentially causes other components to fail(https://en.wiktionary.org/wiki/breaking_change#:~:text=Noun,old%20to%20new%20in%20import).

I don't know 'migration path for existing applications', we can remove it.